tests/tests.py

```diff
@@ -1933,8 +1955,22 @@ def setUp(self):
         self.test_password = rndpassword()
         self.test_groupname = 'testGroupFor_%s' % self.test_manager.project_a
 
-    def test_add_and_remove_user(self):
+    def _log_skip_pycontribs_instance_warning(self, test_name):
+        logging.warning('The current jira admin user for  '
+                        'https://pycontribs.atlassian.net lacks '
+                        'permission to modify users. Therefore, '
+                        'the %s test has been '
+                        'skipped.', test_name)
 
+    def _should_skip_for_pycontribs_instance(self):
+        return self.test_manager.CI_JIRA_ADMIN == 'ci-admin' and (
```

Comment:
@ssbarnea the user needs site administration permissions to make changes to (some) users / groups in cloud.